config/routes.rb

```diff
@@ -3,5 +3,11 @@
     root "static_pages#home"
     get "/home", to: "static_pages#home"
     get "/help", to: "static_pages#help"
+    get "/signup", to: "users#new"
+    post "/signup", to: "users#create"
+    get "/login", to: "sessions#new"
+    post "/login", to: "sessions#create"
+    post "/logout", to: "sessions#destroy"
+    resources :users, only: %i(new create show edit index update destroy)
```

Comment:
nếu dùng hết cả 7 action thì bỏ only đi em
```suggestion
    resources :users
```

Reply:
Ready